common-rendering/src/components/MetaData.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { palette } from "@guardian/src-foundations";
+import type { FC } from "react";
+import { MetaData } from "./MetaData";
+import { Pillar, Theme, Special } from "@guardian/types";
+import { select } from "@storybook/addon-knobs";
+
+const pillarOptions = {
```

Comment:
Feels like this and the `selectPillar` function are being repeated a lot now, could we maybe put them somewhere shared?

Reply:
Yeah, it can do. Again, until there's a clearer picture about this component I'll know more.

Reply:
https://github.com/guardian/dotcom-rendering/pull/3430